src/services/inlineStylePrefixer/prefixData.gen.js

```diff
@@ -0,0 +1,21 @@
+const {
+  'default': generateData,
+  generateFile,
+} = require('inline-style-prefixer/lib/generator');
+
+const browserList = require('../../../config/browsers.json');
+
+module.exports = () => {
+  const browserListForInlineStylePrefixer = {};
+
+  for (const browser in browserList) {
+    if (browser !== 'chromium') {
+      browserListForInlineStylePrefixer[browser] = Number(browserList[browser]);
```

Comment:
Converting versions to a number is throwing a warning for me, saying:

```
Warning, the following targets are using a decimal version:

  safari: 7.1

We recommend using a string for minor/patch versions to avoid numbers like 6.10
getting parsed as 6.1, which can lead to unexpected behavior.```